### PR TITLE
Adding unit tests to Storage-Sample component

### PR DESCRIPTION
**Pull request by DukeDavis12**
Adding unit tests to Storage-Sample components.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

**Review by DukeDavis12 — COMMENTED** [review]
Storage-di-sample code coverage is 80% ( Up from 73%)
Storage-owner-sample code coverage is 82% (Up from 56%)
Storage-rv-sample code coverage is 87% (Up from 42%)

**Review by johneasterday — APPROVED** [review]

File: util/storage-samples/storage-rv-sample/src/test/java/org/fido/iot/storage/RvsStorageTest.java [hunk 1]
```diff
@@ -32,7 +39,9 @@
 import org.fido.iot.protocol.To1ClientStorage;
 import org.fido.iot.protocol.To1ServerService;
 import org.fido.iot.protocol.To1ServerStorage;
+import org.junit.jupiter.api.TestMethodOrder;
 
+@TestMethodOrder(Alphanumeric.class)
```
Comment (Darshini-Parikh93) [inline util/storage-samples/storage-rv-sample/src/test/java/org/fido/iot/storage/RvsStorageTest.java:44]: Could you please help me understand why we need this annotation?

File: util/storage-samples/storage-owner-sample/src/test/java/org/fido/iot/storage/To2StorageTest.java [hunk 2]
```diff
@@ -411,5 +433,391 @@ protected void failed(Exception e) {
...
+    String args[] = new String[]{"-tcp", "-tcpAllowOthers", "-ifNotExists", "-tcpPort", DB_PORT};
+    // start the TCP Server
+    Server server = null;
+
+    try {
+
+      server = Server.createTcpServer(args).start();
+      OwnerDbManager dbsManager = new OwnerDbManager();
+      assertDoesNotThrow(
+          ()-> { dbsManager.addDeviceTypeCriteria(ds, "default",
```
Comment (Darshini-Parikh93) [inline util/storage-samples/storage-owner-sample/src/test/java/org/fido/iot/storage/To2StorageTest.java:626]: Is this a positive scenario where you are trying to add a valid device type criteria?

File: util/storage-samples/storage-rv-sample/src/test/java/org/fido/iot/storage/RvsStorageTest.java (hunk 1 above)
Comment (DukeDavis12) [reply util/storage-samples/storage-rv-sample/src/test/java/org/fido/iot/storage/RvsStorageTest.java:44]: To execute the tests in Alphanumeric order.

File: util/storage-samples/storage-owner-sample/src/test/java/org/fido/iot/storage/To2StorageTest.java (hunk 2 above)
Comment (DukeDavis12) [reply util/storage-samples/storage-owner-sample/src/test/java/org/fido/iot/storage/To2StorageTest.java:626]: Yes.

**Review by johneasterday — APPROVED** [review]

**Review by Darshini-Parikh93 — APPROVED** [review]

**Review by trbehera — APPROVED** [review]

**Pull request merged**